winui/BusyIndicator/AnimationType.md

```diff
@@ -1,12 +1,36 @@
 ---
 layout: post
-title: Content with WinUI BusyIndicator control | Syncfusion
-description: Learn here all about Seven different animation types and features with Syncfusion WinUI BusyIndicator Control.
+title: AnimationType in WinUI BusyIndicator control | Syncfusion
+description: Learn here all about different animation types available in the Syncfusion WinUI BusyIndicator control.
```

Comment:
description: Learn all about different animation types available in the Syncfusion WinUI BusyIndicator control here.

Reply:
Changed the content as mentioned above.